app/graphql/graphql_devise/mutations/login.rb

```diff
@@ -1,34 +1,59 @@
 module GraphqlDevise
   module Mutations
-    class Login < GraphQL::Schema::Mutation
+    class Login < Base
       argument :email,    String, required: true
       argument :password, String, required: true
 
       field :authenticable, GraphqlDevise::Types::AuthenticableType, null: true
+      field :success,       Boolean,                                 null: false
+      field :errors,        [String],                                null: false
```

Comment:
I think this breaks graphql standard on how to return errors. The client shouldn't request them, they shuold be returned automatically